lib/plug/parsers.ex

```diff
@@ -193,7 +193,7 @@ defmodule Plug.Parsers do
       {"content-type", ct} ->
         case Conn.Utils.content_type(ct) do
           {:ok, type, subtype, headers} ->
-            reduce(conn, Keyword.fetch!(opts, :parsers), type, subtype, headers, opts)
+            reduce(conn, Keyword.fetch!(opts, :parsers), {type, subtype, headers}, opts)
```

Comment:
This feels very artificial restriction, it raises more questions than help, and we don't really solve issue of having many arguments.